### PR TITLE
73 editdeletecontacts closes #73 closes #74

### LINKED ISSUE #73: create edit and delete for contacts (3)

### LINKED ISSUE #74: Create delete for students (2)

### PR DESCRIPTION
**Pull request by ChuckPierce**
I have a pull request here.  It closes two issues.  Please let me know what you think.  There is a bunch of repeated code that I think I could optimize somehow but right now this is what I got.

### CODE REVIEW

**Comment by ChuckPierce** [comment]
I created the delete button as well

**Comment by ChuckPierce** [comment]
don't merge this...apparently its still not working...

**Comment by Meliis** [comment]
going through this now, one early thought--what if someone for some reason had one student in two of their classes?  would be a problem if trying to delete them from one class also made their information unavailable for the other.  maybe this doesn't actually happen though

File: client/app/account/settings/newClassroom/studentForm/studentForm.html [hunk 1]
```diff
@@ -11,11 +11,20 @@
...
 		<h4>Contacts</h4>
-		<li ng-repeat="contact in currentStudent.contacts">{{ contact.name }} ({{ contact.relationship }}): {{contact.phone}}</li>
-	</ul>
-	<div class="contact-add-form">
+		<div ng-repeat="contact in currentStudent.contacts">
+			<span ng-hide="addEditView">
+			{{ contact.name }} ({{ contact.relationship }}): {{contact.phone | tel}}
+			<div class="btn-group">
+				<button class="btn btn-default btn-xs" ng-click="addShowEdit(contact._id)"><i class="fa fa-pencil"></i></button>
+				<button class="btn btn-danger btn-xs" ng-click="deleteContact(contact._id)">x</button>
```
Comment (Meliis) [inline client/app/account/settings/newClassroom/studentForm/studentForm.html]: you can use <span class="glyphicon glyphicon-remove"></span>

**Comment by Meliis** [comment]
omg this is so much, good work.  sorry for being so feckless lately :(

**Comment by ChuckPierce** [comment]
thanks and no no I just am already seeing time fly by so fast so I want to get as much as I can done.  Your point makes sense about the students.  If we wanted to have that I think users would have to have students AND classrooms?  Just wondering what would happen if all the classrooms that a student belongs to gets deleted.  Don't know how we would then find them again...this is just my thinking.  But if teachers had students that belonged to them then they could assign students to a classroom if they don't have one and multiple classrooms if needed.

**Comment by ChuckPierce** [comment]
but we can discuss this tomorrow before this is merged in

**Comment by Meliis** [comment]
having users have students is starting to make sense to me.  i feel like we may have to move from there being a "settings" button in the navbar to like "admin" where users can see their classrooms, their students, their unverified contacts, and possibly their scheduled messages (to complement andrew's cart-like quick view).  our hope would be that it's not a page they would have to access very much, but it sounds like a thing.

**Pull request merged**